core/const.py

```diff
@@ -825,4 +827,16 @@ class ITLRegion(StrEnum):
     "PostMitigatedImpact": ("Post-Mitigated Impact", "Programme / Project Risks"),
     "PostMitigatedLikelihood": ("Post-mitigated Likelihood", "Programme / Project Risks"),
     "Proximity": ("Proximity", "Programme / Project Risks"),
+    "Start Date": ("Start Date", "Projects Progress Summary"),
+    "Completion Date": ("Completion Date", "Projects Progress Summary"),
+    "Date of Most Important Upcoming Comms Milestone (e.g. Dec-22)": (
+        "Date of Most Important Upcoming Comms Milestone",
+        "Projects Progress Summary",
+    ),
+    "Private Sector Funding Required": ("Private Sector Funding Required", "Private Sector Investment"),
+    "Private Sector Funding Secured": ("Private Sector Funding Secured", "Private Sector Investment"),
+    "Spend for Reporting Period": ("G-Y", "Project Funding Profiles"),
+    "Amount": ("E-W", "Project Outputs"),
```

Comment:
I don't think "E-W" or "G-Y" are very useful. They're far too vague, and the letters in the spreadsheet to represent the X axis shouldn't be used interchangeably with the actual column names or else it gets confusing (i.e. "Column 'Unit of Measurement' is null" and "Columns 'E-W' are the wrong type" shouldn't really exist in the same validation payload as the use of 'Column' is different in each case). 

This honestly might be a question for design as it's very hard to capture the multiplicity of columns in one generic error message. For now something like, "In columns for Financial Years XXXX-XXX" or "Columns H1 (XXX-XXX), H2 (XXX-XXX) and TOTAL" adheres to our current usage of the word "Column". I admit this is also a suboptimal solution, hence getting design folk involved might be advisable.

Reply:
I agree, I'll try and engage someone from design on this point, but for now will make the change you suggested